src/inim.nim

```diff
@@ -50,10 +65,8 @@ var
 
 when promptHistory:
     # When prompt history is enabled, we want to load history
-    # TODO: Config to allow per-session history
-    var historyFile =  getConfigDir() / "inim"
-    discard existsorCreateDir(historyFile)
-    historyFile = historyFile / ".history"
+    var historyFile = if config.getSectionValue("History", "persistent") == "True": ConfigDir / "history"
```

Comment:
What do you think about calling the file `history.nim`? It mostly contains Nim code, right? Might make it nicer to syntax highlight when reading.

Reply:
Good call. If we want external editor support down the track, it's likely going to be reading that file

Reply:
:ok_hand: